v3/src/models/data/data-set.ts

```diff
@@ -306,7 +306,11 @@ export const DataSet = V2Model.named("DataSet").props({
   get itemsNotSetAside() {
     return self._itemIds.filter(itemId => !self.isItemSetAside(itemId))
   },
-  // ids of items that have not been hidden (set aside) or filtered by user
+  /**
+   * ids of items that have not been hidden (set aside) or filtered by user
+   * Note: this will not take into account any filtering applied at the visualization
+   * level: visualization hidden cases, visualization filter formula, or displayOnlySelectedCases
+   */
```

Comment:
I don't think this comment change is warranted. The DataSet shouldn't have any knowledge of or reference to how clients use the data. This adds CODAP-specific lingo to a module that is (at least theoretically) not CODAP-specific.